Samples/Swift/DaysUntilBirthday/macOS/DaysUntilBirthdayOnMac.entitlements

```diff
@@ -8,5 +8,7 @@
 	<true/>
 	<key>com.apple.security.network.client</key>
 	<true/>
+	<key>keychain-access-groups</key>
```

Comment:
Did you mean to add this?

Reply:
Yes, the sample won't run without it on macOS. I wonder if you see the same?

Reply:
Are you signing the build on macOS?

Reply:
I've removed this change and created #192 to track this issue.

Reply:
I'm still seeing the change in this PR.

Reply:
Hm, maybe it wasn't reverted before? In any case, it's no longer in this PR.